engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/build/EspProcessBuilder.scala

```diff
@@ -21,8 +21,8 @@ class ProcessMetaDataBuilder private[build](metaData: MetaData) {
   def subprocessVersions(subprocessVersions: Map[String, Long]) =
     new ProcessMetaDataBuilder(metaData.copy(subprocessVersions = subprocessVersions))
 
-  def additionalFields(userDefinedProcessAdditionalFields: UserDefinedProcessAdditionalFields) =
-    new ProcessMetaDataBuilder(metaData.copy(additionalFields = Some(userDefinedProcessAdditionalFields)))
+  def additionalFields(processAdditionalFields: ProcessAdditionalFields) =
```

Comment:
I know that it already has been done this way, but maybe better approach would be pass here all fields of `ProcessAdditionalFields` instead of `ProcessAdditionalFields` object. Now it doesn't look too much DSL-y. What do you think about that?

Reply:
with default parameter values it seems reasonable to me